src/current/v23.2/alter-database.md

```diff
@@ -834,6 +836,120 @@ The `lease_preferences` field is now updated to include `us-west1`.
 
 To remove the changes made in this example, [reset the Zone Config Extensions](#reset-a-regions-zone-config-extensions).
 
+#### Implement super regions
+
+In this example, we use [Zone Config Extensions]({% link {{ page.version.version }}/zone-config-extensions.md %}) to provide an alternative implementation of [Super Regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions), which are useful for [data domiciling]({% link {{ page.version.version }}/data-domiciling.md %}).
+
+For this example, you need to start a [`cockroach demo` cluster]({% link {{ page.version.version }}/cockroach-demo.md %}) using the following command, which [defines the available cluster localities using the `--demo-locality` flag]({% link {{ page.version.version }}/cockroach-demo.md %}#demo-locality):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --nodes=9 --demo-locality=region=us-east1:region=us-east1:region=us-central1:region=us-west1:region=europe-west1:region=europe-west1:region=europe-central1:region=europe-east1:region=europe-east1
+~~~
+
+Next, enter the statements below at the SQL prompt. These statements will:
+
+- Make the [`movr` database]({% link {{ page.version.version }}/movr.md %}) into a [multiregion database]({% link {{ page.version.version }}/multiregion-overview.md %}) by [setting the primary region]({% link {{ page.version.version }}/alter-database.md %}#set-primary-region).
+- Set the `movr` database to [survive region failures]({% link {{ page.version.version }}/multiregion-survival-goals.md %}#survive-region-failures).
+- Add the remaining regions we defined at cluster start time to the multi-region `movr` database (for a total of 6).
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr SET PRIMARY REGION "us-east1";
+ALTER DATABASE movr SURVIVE REGION FAILURE;
```

Comment:
The `ALTER DATABASE movr SURVIVE REGION FAILURE;` will need to go after the `ADD REGION` commands.

Reply:
Thanks, this is updated in the latest commit